src/Http/ApiProblem/ApiProblem.php

```diff
@@ -562,4 +562,14 @@ public static function failedToSubscribeToNewsletter(string $detail): self
             $detail
         );
     }
+
+    public static function cannotDeleteUitpasPlace(): self
+    {
+        return self::create(
+            'https://api.publiq.be/probs/uitdatabank/cannot-delete-uitpas-place',
+            'Cannot delete UiTPAS place',
+            StatusCodeInterface::STATUS_FORBIDDEN,
+            'Place is an UiTPAS balie. UiTPAS locations cannot be deleted.'
```

Comment:
There is a naming mix of place and location

Reply:
Fixed it. Also replaced the word "balie" which is dutch.